debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java

```diff
@@ -92,6 +93,7 @@
                                                      .field(LSN_KEY, Schema.OPTIONAL_INT64_SCHEMA)
                                                      .field(SNAPSHOT_KEY, Schema.OPTIONAL_BOOLEAN_SCHEMA)
                                                      .field(LAST_SNAPSHOT_RECORD_KEY, Schema.OPTIONAL_BOOLEAN_SCHEMA)
+                                                     .field(LAST_EVENT_FOR_LSN, Schema.OPTIONAL_BOOLEAN_SCHEMA)
```

Comment:
This is causing that field to show up in emitted messages, which is not desirable as it's just an internal detail which we shouldn't expose. I can't see an easy alternative, though, other than not putting the source records themselves into the blocking queue instead of a wrapper which contains the source record and, separately from that, all internally needeed metadata such as this flag. Any thoughts?

Reply:
Hmm, how about extending `SourceRecord` and add to flag there? So intenally we will use PostgresSourceRecord and can add arbitrary metadata to it?

Reply:
Not sure whether sub-classing would work, won't the extra field still show up in the outgoing message. Also it seems hacky to me, I'd prefer a composition:

```
class ChangeEvent {
    SourceRecord record;
    boolean isLastOfLsn;
}
```

Question is, how is doing it :)

Reply:
It will definitely not show up as the value that is serialized and sent over the wire is passed as value in `SourceRecord` constructor parameter.

Reply:
I'm not following, I saw the field in the output in the console consumer.

Reply:
It will not show up if we use the inheritance approach. Right now it is true it is present.